packages/remote-config/lib/index.js

```diff
@@ -116,36 +184,73 @@ class FirebaseConfigModule extends FirebaseModule {
   }
 
   get isDeveloperModeEnabled() {
-    return this._isDeveloperModeEnabled;
+    console.warn(
+      'firebase.remoteConfig().isDeveloperModeEnabled has now been removed. Please consider setting `settings.minimumFetchIntervalMillis` in remoteConfig.Settings',
+    );
   }
 
   get minimumFetchInterval() {
-    return this._minimumFetchInterval;
+    console.warn(
+      'firebase.remoteConfig().minimumFetchInterval has now been removed. Please consider setting `settings.minimumFetchIntervalMillis` in remoteConfig.Settings',
+    );
   }
 
-  setConfigSettings(settings = {}) {
-    if (!isObject(settings) || !hasOwnProperty(settings, 'isDeveloperModeEnabled')) {
-      throw new Error(
-        "firebase.remoteConfig().setConfigSettings(): 'settings' must be an object with a 'isDeveloperModeEnabled' key.",
-      );
+  /**
+   * Deletes all activated, fetched and defaults configs and resets all Firebase Remote Config settings.
+   * @returns {Promise<null>}
+   */
+  reset() {
+    if (isIOS) {
+      console.warn('firebase.remoteConfig().reset() is not yet supported on iOS.');
```

Comment:
We don't normally warn if a method isn't cross platform, user should be able to call the method without having to write platform specific code to ignore the warnings.